insight/tests/tasks/weekly_user_trend/conftest.py

```
@@ -0,0 +1,10 @@
+from datetime import datetime
```

Ruff (0.12.2)

`datetime.datetime` imported but unused

Remove unused import: `datetime.datetime`

(F401)